src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface Drawable {
+
+    void drawFigure();
```

Comment:
Why drawFigure ? I do not see any Figures here.

```suggestion
    void draw();
```